packages/integration-karma/test/swapping/styles/index.spec.js

```diff
@@ -0,0 +1,24 @@
+import { createElement } from 'lwc';
+
+// TODO [#1869]: getting the global API from global LWC in tests until it is allowed in compiler
+const { swapStyle } = LWC;
+
+import Simple from 'base/simple';
+const { blockStyle, inlineStyle, noneStyle } = Simple;
+
+describe('style swapping', () => {
+    it('should work with components with implicit style definition', () => {
+        const elm = createElement('x-simple', { is: Simple });
+        document.body.appendChild(elm);
+        expect(getComputedStyle(elm.shadowRoot.firstChild).display).toBe('block');
+        swapStyle(blockStyle[0], inlineStyle[0]);
```

Comment:
@diervo what I noticed is that when you import a css file, what you get is the array, not the factory function. That's why I have to use `[0]` everywhere here... not ideal. The question is, how does your other part work? is it an array, or each individual one? we can certainly do the thing on the array identity from the import instead of individual style factories.